cms/djangoapps/contentstore/models.py

```diff
@@ -308,13 +343,22 @@ def filter_links(
     ) -> QuerySet["EntityLinkBase"]:
         """
         Get all links along with sync flag, upstream context title and version, with optional filtering.
+
+        `use_top_level_parents` is an special filter, replace any result with the top-level parent if exists.
+        Example: We have linkA and linkB with top-level parent as linkC and linkD without top-level parent.
+        After all other filters:
+        Case 1: `use_top_level_parents` is False, the result is [linkA, linkB, linkD]
```

Comment:
```suggestion
        Case 1: `use_top_level_parents` is False, the result is [linkA, linkB, linkC, linkD]
```

It should list all components no?

Reply:
Updated: https://github.com/openedx/edx-platform/pull/37076/commits/bc484ec4b1bb88047cd8782f281594470ae1bc67

Reply:
@ChrisChV Just want to confirm whether `linkC` will not show up if `use_top_level_parents` is False?